tasks/app/run.py

```diff
@@ -35,10 +38,6 @@ def warmup(
     """
     Pre-configure the Houston API Server before running
     """
-    # Automatically use the production config when running a public web server
-    if host in ['0.0.0.0'] and flask_config is None:
```

Comment:
Did we remove the conditional check where `flask_config is None`?

Reply:
This is the only check I see for that in this logic. But the main creation logic checks it a few different times as well. So I'd say we're covered. In fact I see a weird bit of logic around that that I'll followup on later.

Reply:
In other words, an assertion would raise if the flask_config isn't set.